src/content/en/fundamentals/performance/speed-tools/images/tool-psi.svg

```diff
@@ -0,0 +1 @@
nUse"><stop offset="0" stop-color="#fff" stop-opacity=".1"/><stop offset="1" stop-color="#fff" stop-opacity="0"/></radialGradient></defs><title>icon_pagespeed-insights_256px</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><circle id="_Path_" data-name="&lt;Path&gt;" class="cls-1" cx="145.51" cy="145.51" r="145.51"/><g id="_Group_" data-name="&lt;Group&gt;"><g id="_Group_2" data-name="&lt;Group&gt;"><path id="_Path_2" data-name="&lt;Path&gt;" class="cls-2" d="M237.61 91.19v121.2a12.53 12.53 0 0 1-12.54 12.54H66.26a12.53 12.53 0 0 1-12.54-12.54V91.19z"/><path id="_Path_3" data-name="&lt;Path&gt;" class="cls-3" d="M66.26 66.11h158.81a12.53 12.53 0 0 1 12.54 12.54v12.54H53.73V78.65a12.53 12.53 0 0 1 12.53-12.54z"/><path id="_Path_4" data-name="&lt;Path&gt;" class="cls-4" d="M225.07 66.11H66.26a12.57 12.57 0 0 0-12.53 12.54v1a12.57 12.57 0 0 1 12.53-12.5h158.81a12.57 12.57 0 0 1 12.54 12.54v-1a12.57 12.57 0 0 0-12.54-12.58z"/><path id="_Path_5" data-name="&lt;Path&gt;" class="cls-5" d="M53.73 91.19h183.88v1H53.73z"/><path id="_Path_6" data-name="&lt;Path&gt;" class="cls-6" d="M145.6 128.8a66.77 66.77 0 0 0-66.76 66.87c0 1.4-.08 3.13 0 4.18H108a37 37 0 0 1 0-4.18 37.64 37.64 0 0 1 57.36-32l21-21a66.6 66.6 0 0 0-40.76-13.87z"/><path id="_Path_7" data-name="&lt;Path&gt;" class="cls-7" d="M186.32 142.63l-21 21a37.59 37.59 0 0 1 17.91 32 37 37 0 0 1 0 4.18h29.26c.08-1 0-2.78 0-4.18a66.73 66.73 0 0 0-26.17-53z"/><circle id="_Path_8" data-name="&lt;Path&gt;" class="cls-8" cx="70.44" cy="78.65" r="4.18"/><circle id="_Path_9" data-name="&lt;Path&gt;" class="cls-8" cx="87.16" cy="78.65" r="4.18"/><path id="_Path_10" data-name="&lt;Path&gt;" class="cls-5" d="M225.09 223.87H66.22a12.57 12.57 0 0 1-12.54-12.54v1a12.57 12.57 0 0 0 12.54 12.54h158.87a12.57 12.57 0 0 0 12.54-12.54v-1a12.57 12.57 0 0 1-12.54 12.54z"/><path id="_Path_11" data-name="&lt;Path&gt;" class="cls-9" d="M186.54 145.17a2.06 2.06 0 0 0 .4-1.22 2.1 2.1 0 0 0-2.09-2.09 2.05 2.05 0 0 0-1.29.46L138 177.39a12.53 12.53 0 1 0 18.37 16.41l30.16-48.65z"/><path id="_Path_12" data-name="&lt;Path&gt;" class="cls-10" d="M138 178.44l45.56-35.08a2.06 2.06 0 0 1 3.29 1.09 1.94 1.94 0 0 0 .08-.5 2.1 2.1 0 0 0-2.09-2.09 2.05 2.05 0 0 0-1.29.46L138 177.38a12.48 12.48 0 0 0-4.88 9.92v.56a12.52 12.52 0 0 1 4.88-9.42z"/><path id="_Path_13" data-name="&lt;Path&gt;" class="cls-11" d="M186.54 145.17l-30.16 48.65a12.52 12.52 0 0 1-23.22-5.93v.48a12.53 12.53 0 0 0 23.25 6.5l30.16-48.65a2.06 2.06 0 0 0 .4-1.22 1.85 1.85 0 0 0-.08-.54 2.22 2.22 0 0 1-.35.71z"/><path id="_Path_14" data-name="&lt;Path&gt;" class="cls-12" d="M186.51 142.69a2.06 2.06 0 0 1 0 2.48l-30.16 48.65a12.52 12.52 0 0 1-18.84 3l28.07 28.07h59.47a12.53 12.53 0 0 0 12.54-12.54v-18.56z"/></g><path id="_Path_15" data-name="&lt;Path&gt;" class="cls-13" d="M237.61 78.65a12.53 12.53 0 0 0-12.54-12.54H66.26a12.53 12.53 0 0 0-12.53 12.54v133.73a12.53 12.53 0 0 0 12.54 12.54h158.8a12.53 12.53 0 0 0 12.54-12.54z"/></g></g></g></svg>
```

Comment:
please check if this icon exists anywhere else on the site, and use that if possible.